doc/source/t-compsets.rst

```diff
@@ -19,11 +19,12 @@ could within the coupled system.
 
 A run with standalone CISM in the CESM context is known as a T compset. (In the past this
 was referred to as a TG compset. Now the G appears near the end of the compset name, so
-compsets have names like T1850G.)  This compset uses the active ice sheet model forced by
-a data land model; all other components are stubs. Before running a T compset, you must
-have coupler history files from a previous run that included CLM (version 4.5 or
-later). You can run with either existing forcing data (see :numref:`t-with-existing-data`)
-or with your own forcing data (see :numref:`t-with-your-own-data`).
+compsets have names like T1850Gg, where the final g indicates Greenland.) This compset
+uses the active ice sheet model forced by a data land model; all other components are
```

Comment:
"This compset uses the active ice sheet model for the specified ice sheet(s); all other components...."

Reply:
@Katetc I have added your suggestion, but also kept "forced by a data land model" (were you suggesting that I should get rid of that?)